examples/depthEstimation-single-image/sketch.js

```diff
@@ -0,0 +1,137 @@
+/*
+ * 👋 Hello! This is an ml5.js example made and shared with ❤️.
+ * Learn more about the ml5.js project: https://ml5js.org/
+ * ml5.js license and Code of Conduct: https://github.com/ml5js/ml5-next-gen/blob/main/LICENSE.md
+ *
+ * This example demonstrates depth estimation on a single image using ml5.depthEstimation.
+ */
+
+// Global variables
+let depthEstimator; // The ml5.js depth estimation model instance
+let img;            // The p5.js image object for the input
+let depthResult;    // The object holding the depth estimation results
+let status = "Loading model and image..."; // Status message
+
+// Image dimensions (will be updated based on loaded image)
+let imgWidth = 640; // Default width
+let imgHeight = 480; // Default height
+
+// Model and runtime options
+let options = {
+    colormap: "COLOR", // Colormap for visualization ('COLOR', 'GRAYSCALE')
+    minDepth: 0,       // Minimum depth value (0-1) for normalization
+    maxDepth: 1,       // Maximum depth value (0-1) for normalization
+};
+
+// We don't use preload anymore, setup will handle async loading
+// function preload() {}
+
+async function setup() {
+    // Create a canvas twice the width of the image to show image and depth map side-by-side
+    // Initial size is default, will be resized later
+    createCanvas(imgWidth * 2, imgHeight);
+    console.log("Setup started.");
+
+    // Initialize the depth estimation model
+    console.log("Loading depth estimation model...");
+    depthEstimator = ml5.depthEstimation(options);
+
+    // Load the sample image using a promise wrapper for await
+    console.log("Loading image...");
+    try {
+        // --- IMPORTANT: Replace with the path to YOUR image ---
+        img = await loadImagePromise('face.png'); // Use the user's image
+        // img = await loadImagePromise('../../3d_photo/images/im1.jpg'); // Original image path
```

Comment:
Since the `face.png` is a file in the current folder, spelling those steps out in such detail might be unnecessary